src/filesystem_sdl.cpp

```diff
@@ -114,11 +171,31 @@ static size_t SDLCALL ifs_read(struct SDL_RWops *context, void *ptr, size_t size
 
 	return static_cast<int>(num);
 }
+static size_t SDLCALL ofs_read(struct SDL_RWops * /*context*/, void * /*ptr*/, size_t /*size*/, size_t /*maxnum*/) {
+	SDL_SetError("Reading not implemented");
+	return 0;
+}
 
 static size_t SDLCALL ifs_write(struct SDL_RWops * /*context*/, const void * /*ptr*/, size_t /*size*/, size_t /*num*/) {
 	SDL_SetError("Writing not implemented");
 	return 0;
 }
+static size_t SDLCALL ofs_write(struct SDL_RWops *context, const void *ptr, size_t size, size_t num) {
+	std::ostream *ofs = static_cast<std::ostream*>(context->hidden.unknown.data1);
+
+	const std::streampos before = ofs->tellp();
+	ofs->write(static_cast<const char*>(ptr), num * size);
+	const std::streampos after = ofs->tellp();
+
+	const int written = (after - before) / size;
+
+	// Fail sticks unless we clear it. Bad is an actual I/O error
+	if(!ofs->bad())
+		ofs->clear();
```

Comment:
Please remove this part. `std::basic_ostream::write()` isn't supposed to set failbit, ever.

> Also, unlike the formatted output functions, this function does not set the failbit on failure.

http://en.cppreference.com/w/cpp/io/basic_ostream/write